### ISSUE TITLE
Make views updatable

### ISSUE THREAD

**Issue by chattago2002** [mevdschee/php-crud-api#918]
Hi.

I read that at the moment views aren't updatable (insert, update, delete) as also reported in the #823 question [here](https://github.com/mevdschee/php-crud-api/issues/823#issuecomment-968661002).
Obviously when I try to update a view the response is **1015 error** with message **Operation 'update' not supported**.

As reported in last comment of #823 views seems to be updated bypassing the check.
Is there the possibility to add an option (or something else) in the api configuration file to let this operation if necessary? 

P.S. I'm using MySQL instead of SQL Server but this is not a difference.

Thank you

**Comment by mevdschee** [comment]
Yeah, the primary key reflection does not work, nor can all views be updated (only update-able views). To avoid support tickets I disabled the functionality. Although indeed we could allow it to be enabled with a configuration option, which is not a bad idea.
